labels.js

```diff
@@ -0,0 +1,29 @@
+// Flatten the labels defined in the config into a single array of objects, as
+// expected by github-label-sync:
+// 
+// [
+//   {
+//     "name": "mylabel",
+//     "color": "ff0000",
+//     "description": "Description of the label",
+//     "aliases": []
+//   },
+//   ...
+// ]
+
+module.exports = require ('./config.js').reduce((accumulator, group) => {
```

Comment:
Is it possible to avoid reduce and do something less clever?

Interesting thread on ways to avoid reduce: https://twitter.com/jaffathecake/status/1213077702300852224

Reply:
We have access to `flat` and `flatMap` for example.

Reply:
```javascript
require ('./config.js').flatMap(group => {
 return group.labels.map(entry => {
        if (typeof entry == 'string') {
            return {
                name: entry,
                color: group.color
            }
        } else {
            // Merge the object with the colour from the group, with the label
            // colour taking precedence.
            return Object.assign({
                color: group.color
            }, entry)
        }
    }
})
```

Reply:
not tested it, if it doesnt work, think there may be a more verbose and boring way to do this.

Reply:
We can use flat and flatMap if we bump to using node 11 / 12 for this repo? Otherwise, I can look at replacing it with a loop.

Reply:
Pushed a revised version that uses `forEach`